.github/workflows/app-release.yml

```diff
@@ -0,0 +1,115 @@
+name: 'Tauri GUI Release'
+on:
+  push:
```

Comment:
Maybe consider running the action only when the [app directory is modified](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-including-paths)...

Reply:
it's just a test this is a release workflow so it will only be run on release